test/integration/targets/meraki_network/tasks/main.yml

```diff
@@ -231,11 +242,11 @@
   - name: Present assertions
     assert:
       that:
-        - create_net_no_type.status == 500
+        # - create_net_no_type.status == 500
```

Comment:
These appear to match earlier tests... can these be removed or uncommented?

Reply:
Tests are updated to take this into account.